HW1/P6/P6.py

```diff
@@ -0,0 +1,72 @@
+import findspark
+findspark.init()
+import pyspark
+import time
+import re
+from random import random
+from bisect import bisect
+start = time.time()
+sc = pyspark.SparkContext(appName='test')
+
+def mapFunc(word):
```

Comment:
It may be helpful to define these functions in the middle of your code when they are needed. Defining them at the beginning makes it difficult to understand when you will use them.